macintyrej7/CMPT305_GroupProject
# Hooked up Map Screen Slider.

Test Procedure:
1)Click on a point, the default radius is 0.5.
2)Modify Radius to be larger, hit apply, click on same school.
3) Reset filter criteria and see that the radius is again 0.5KM, click on same school
